cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -126,6 +122,135 @@ impl LockedManifest {
         Ok(ContainerBuilder::new(container_builder_path))
     }
 
+    pub fn read_from_file(path: &CanonicalPath) -> Result<Self, LockedManifestError> {
+        let contents = fs::read(path).map_err(LockedManifestError::ReadLockfile)?;
+        serde_json::from_slice(&contents).map_err(LockedManifestError::ParseLockfile)
+    }
+
+    pub fn check_lockfile(
```

Comment:
question: shouldn't this be on `LockedManifestPkgdb`?

Reply:
i guess with the current interface, and considering we dont yet have the equivalent, yes.